### PR TITLE
test: [M3-6620] - Add assertions for created LKE cluster in Cypress LKE tests

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
Assertions should be added to the [LKE creation e2e test](https://github.com/linode/manager/blob/develop/packages/manager/cypress/e2e/kubernetes/lke-create.spec.ts) to confirm that the correct information is shown

## Changes  🔄
- Add assertions for created LKE cluster in Cypress LKE tests

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/kubernetes/lke-create.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts [hunk 1]
```diff
@@ -114,6 +115,11 @@ describe('LKE Cluster Creation', () => {
 
     cy.get('[data-testid="ha-radio-button-yes"]').should('be.visible').click();
 
+    let totalcpu = 0;
+    let totalmemory = 0;
+    let totalstorage = 0;
+    let monthprice = 0;
```
Comment (dwiley-akamai) [inline packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts:121]: Let's camel case these to be consistent with how we typically handle variable names

File: packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts [hunk 2]
```diff
@@ -150,7 +156,29 @@ describe('LKE Cluster Creation', () => {
...
+        totalmemory = totalmemory + nodeCount * 4;
+        totalstorage = totalstorage + nodeCount * 80;
+        monthprice = monthprice + nodeCount * 24;
+      }
+      if (clusterPlan.size == 4 && clusterPlan.type == 'Dedicated') {
+        totalcpu = totalcpu + nodeCount * 2;
+        totalmemory = totalmemory + nodeCount * 4;
+        totalstorage = totalstorage + nodeCount * 80;
+        monthprice = monthprice + nodeCount * 36;
+      }
```
Comment (dwiley-akamai) [inline packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts:178]: On first glance I found these a bit unclear. The four variables are initialized to `0` and don't get modified anywhere else, right? Couldn't we just do, for example, `totalStorage = nodeCount * 50`?

**Review by dwiley-akamai — APPROVED** [review]
Confirmed the test passes remotely ✅

<details>
<summary>(I did observe a failure locally)</summary>

Screenshot 2024-06-18 at 5 18 48 PM

</details>

File: packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts (hunk 1 above)
Comment (AzureLatte) [reply packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts:121]: Nice catch! I will update them

File: packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts (hunk 2 above)
Comment (AzureLatte) [reply packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts:178]: This is for the scenario when different linodes selected, the values can be calculated as a total. For example, when both ```clusterPlan.size == 2 && clusterPlan.type == 'Linode'``` and ```clusterPlan.size == 4 && clusterPlan.type == 'Dedicated'``` picked, the four variables need to be added based on the previous sum.

**Review by cpathipa — APPROVED** [review]

**Pull request merged**